core/src/androidTest/java/novoda/lib/sqliteprovider/analyzer/DatabaseAnalyzerTest.java

```diff
@@ -0,0 +1,126 @@
+package novoda.lib.sqliteprovider.analyzer;
+
+import android.database.sqlite.SQLiteDatabase;
+import android.test.AndroidTestCase;
+import android.test.MoreAsserts;
+import android.test.RenamingDelegatingContext;
+import android.text.TextUtils;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static android.database.DatabaseUtils.createDbFromSqlStatements;
+
+public class DatabaseAnalyzerTest extends AndroidTestCase {
```

Comment:
Copied and modified from `DBUtilsTest` as `DatabaseAnalyzer` was copied and modified from `DBUtils`